cpu/sam0_common/include/periph_cpu_common.h

```diff
@@ -670,6 +670,218 @@ typedef struct {
  */
 #define WDT_HAS_INIT                   (1)
 
+/**
+ * The sam0 DMA peripheral has a number of channels. Each channel is a separate
+ * data stream, triggered by a configurable trigger when enabled, or triggered
+ * by software (not yet supported). In theory each DMA channel is equal and can
+ * have a configurable priority and can be triggered by the full set of triggers
+ * available.
+ *
+ * DMA descriptors, specifying a single transfer with size, source and
+ * destination, are kept in RAM and are read when the channel is enabled and
+ * triggered. On the SAML platform, these descriptors must reside in the LP
+ * SRAM.
+ */
+
+/**
+ * @brief Indicates that the peripheral doesn't utilize the DMA controller.
+ *        Matches with the register configuration for software based triggers.
+ */
+#define DMA_TRIGGER_DISABLED           0
+
+/**
+ * @brief Move the DMA descriptors to the LP SRAM. Required on the SAML21 and
+ * SAML1X families
+ */
+#if defined(CPU_FAM_SAML21) || defined(CPU_FAM_SAML1X)
```

Comment:
SAML1x doesn't have LP RAM

Reply:
Guess I assumed a bit too blindly that the whole SAML range has LP SRAM :disappointed:

Reply:
Thanks for noticing, Fixed!